### DISCUSSION TITLE
High CPU load after all servers in a disk pool was offline due to network disruptions.

### DISCUSSION THREAD

**Discussion by AlfLjung** [minio/minio#18114] [Q&A] [answered]
Hi,
Our setup:
12 servers (RHEL 8.8) , 24 drives on each server
MinIO release : Version: 2023-08-16T20:17:30Z
Pool config:
   1st, Erasure sets: 12, Drives per erasure set: 16
   2nd, Erasure sets: 6, Drives per erasure set: 16

On Saturday 16th of September, we had a major network disruption and all servers in 2nd disk pool was offline for several hours. When everything was brought back online, when running mc admin info command the 2nd disk pool was not visible, and we then performed mc admin service restart command and the disk pool was back. 

After the minio restart we can see that the 4 server have a constant load of 100% CPU (before the weekend we got an average of 20%)
We have now tried to stop minio on all servers restarted servers and bring it up again but we still see the high load (IO wait is approx 95 - 98 %)

![image](https://github.com/minio/minio/assets/92723318/55bcfb3d-1ac8-46cf-a8ee-cf7834f82323)

We also see a large increase of TCP connections.

![image](https://github.com/minio/minio/assets/92723318/f633dad7-c750-4e8d-9a4f-780d826738ad)

We have now pause all client traffic but we still have same CPU load. So I'm guessing that the minio cluster is generating this load ?
I have run mc admin heal command to see if there are any active healing ongoing but output says that there are no healing ongoing..

$ /home/xxxxxx/RELEASE.2023-08-16T20-17-30Z/mc admin heal xxxxxxx/
No active healing is detected for new disks.

I'm out of ideas do you have any suggestion what next step is to resolve this CPU load issue ?

Regards
/Alf

**Comment by klauspost** [comment, marked as answer]
@AlfLjung We only support latest release outside of [SUBNET](https://min.io/pricing) as there are many [releases since this](https://github.com/minio/minio/releases).

To diagnose this we would need server profiles investigation which we similarly only offer to subscribers.

**Reply by AlfLjung** [reply]
Thanks for your reply
